.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,29 @@
+name: codeql
+
+# FIXME uncomment conditions before merging PR
+on:
+  push:
+    # branches:
+    # - master
+  pull_request:
+    # branches:
+    # - master
+  # schedule:
+  #   - cron: "0 3 * * 4"
+
+jobs:
+  analyze:
+    runs-on: ubuntu-18.04
```

Comment:
maybe 20.04 or even latest.

Reply:
TBH I don't think it matters. I chose 18.04 as it generally has wider support (still) and avoids the "`ubuntu-latest` will soon default to `ubuntu-20.04`" warnings in GitHub Actions.